.github/workflows/nightly_detect_failure.yml

```diff
@@ -227,6 +227,18 @@ jobs:
         chmod 600 ~/.zuliprc
         echo "Setup complete"
 
+    - name: Clean workspace and checkout Mathlib4
+      if: steps.check_branch.outputs.result == 'false'
+      run: |
+        sudo rm -rf -- *
```

Comment:
I suppose it would be enough to just rm rf the Mathlib directory?

Reply:
I *think* the checkout script just splats the repository into your working dir, not a subdir.